app/src/main/res/layout/activity_settings.xml

```diff
@@ -0,0 +1,70 @@
+<?xml version="1.0" encoding="utf-8"?>
+<LinearLayout xmlns:android="http://schemas.android.com/apk/res/android"
```

Comment:
use coordinator layout and surround it with scroll view. Also implement the scroll behaviour in toolbar.